soroban-env-common/env.json

```diff
@@ -1398,7 +1398,8 @@
                         }
                     ],
                     "return": "Void",
-                    "docs": "If the TTL for the current contract instance and code (if applicable) is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger"
+                    "docs": "If the TTL for the current contract instance and code (if applicable) is below `threshold` ledgers, extend `live_until_ledger_seq` such that TTL == `extend_to`, where TTL is defined as live_until_ledger_seq - current ledger",
+                    "max_supported_protocol": 21
```

Comment:
Correct. You can't do that yet. The `max_supported_protocol` mechanism is there to retire a function if no contract on chain still links to it (maybe gradually over time it can be achieved, but the exact mechanism to determine that is still a bit TBD), and we can't break contracts by forcing a function to retire.

Reply:
Yeah that's true, I completely overlooked the fact that Soroban is already on pubnet 😬 , removing the protocol guard now.